internal/pkg/mapper/branchmetadata/local_save.go

```diff
@@ -16,6 +19,19 @@ func (m *branchMetadataMapper) MapBeforeLocalSave(recipe *model.LocalSaveRecipe)
 		return nil
 	}
 
+	if branch.IsDefault {
```

Comment:
Tohle má problém v tom, že se ten soubor nevytvoří když se nepullne default branch. (Takže v `pull/only-allowed-branch` prázdný `description.md` není.) Nicméně to je vlastně by-design tím že se to ukládá do metadat default branche v pořádku, ne? 🤔

Reply:
jj, to je myslim ok, ak nemas main branch, nemas description (je to skor "by poor design" a chybajuci project endpoint).